test/test_p4est.jl

```diff
@@ -247,6 +325,24 @@ end
             @test all(Ferrite.morton.(tree.leaves,l,5) == collect(1:2^(3*l)))
         end
     end
+
+    # Reproducer test for Fig.3 
```

Comment:
```suggestion
    # Reproducer test for Fig.3 BWG 11
```
since there are already parts which reference the other algorithmic paper. let me know if I can review